### PR TITLE
Env config for HA List YAMLs

### PR DESCRIPTION
**Pull request by JacobJaffe**
- adds YAML configs specific per environment. 
- moves the entirety of the Google Import to a feature flag, to avoid a weird line on the settings screen (cleanup for TestFlight build)

### ❗  BlueTooth Note ❗
I've kept **all** versions of the BT envs pointing at the **staging** yaml. This is so that this PR is non-breaking for any BT usage of the yaml usages.

(All envs _except_ gps release are for staging)

### CODE REVIEW

File: .env.gps.release [hunk 1]
```diff
@@ -5,5 +5,4 @@
 # prepend ENVFILE=.env.gps.release before any build commands.
 
 TRACING_STRATEGY=gps
-
-# flag_google_import=true
+AUTHORITIES_YAML_ROUTE=https://raw.githubusercontent.com/Path-Check/trusted-authorities/master/production/authorities.1.0.1.yaml
```
Comment (JacobJaffe) [inline .env.gps.release:8]: PRODUCTION

File: .env.bt.release [hunk 2]
```diff
@@ -5,3 +5,4 @@
 # prepend ENVFILE=.env.bt.release before any build commands.
 
 TRACING_STRATEGY=bt
+AUTHORITIES_YAML_ROUTE=https://raw.githubusercontent.com/Path-Check/trusted-authorities/master/staging/authorities.1.0.1.yaml
```
Comment (JacobJaffe) [inline .env.bt.release:8]: STAGING

File: app/api/healthcareAuthorities/getHealthcareAuthoritiesApi.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import Yaml from 'js-yaml';
-import { AUTHORITIES_LIST_URL_MVP1 } from '../../constants/authorities';
+import env from 'react-native-config';
```
Comment (johnschoeman) [inline app/api/healthcareAuthorities/getHealthcareAuthoritiesApi.ts:2]: minor nitpick, but what do you think about a line to break the imports from the const?

File: app/views/Settings/index.tsx [hunk 4]
```diff
@@ -142,11 +142,11 @@ const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
         )}
 
         {isGPS ? (
-          <View style={styles.section}>
-            <FeatureFlag name={'google_import'}>
+          <FeatureFlag name={'google_import'}>
```
Comment (johnschoeman) [inline app/views/Settings/index.tsx:145]: 💯

**Review by johnschoeman — APPROVED** [review]
LGTM!

File: app/api/healthcareAuthorities/getHealthcareAuthoritiesApi.ts (hunk 3 above)
Comment (JacobJaffe) [reply app/api/healthcareAuthorities/getHealthcareAuthoritiesApi.ts:2]: agree!

**Pull request merged**